packages/jmix-front-generator/src/building-blocks/stages/answers/pieces/refineAnswers.ts

```diff
@@ -0,0 +1,54 @@
+import {ProjectModel} from "../../../../common/model/cuba-model";
+import {
+  EntityInfo, RestQueryInfo, RestServiceMethodInfo,
+  StudioTemplateProperty,
+  StudioTemplatePropertyType,
+  ViewInfo
+} from "../../../../common/studio/studio-model";
+import {findEntity, findQuery, findServiceMethod, findView} from "../../../../common/model/cuba-model-utils";
+
+/**
+ * This function takes the answers and adds context / additional data retrieved from project model
+ *
+ * @param projectModel
+ * @param generatorParams
+ * @param answers
+ */
+export function refineAnswers<T>(projectModel: ProjectModel, generatorParams: StudioTemplateProperty[], answers: any): T {
```

Comment:
Need more detailed description. Also looks like `refine` name is not fit well here.